browser/brave_content_browser_client.cc

```diff
@@ -38,6 +39,7 @@
 #include "chrome/common/url_constants.h"
 #include "components/content_settings/core/browser/cookie_settings.h"
 #include "components/services/heap_profiling/public/mojom/heap_profiling_client.mojom.h"
+#include "components/version_info/version_info.h"
```

Comment:
is there already a dep for this in BUILD.gn?

Reply:
Oh, thanks for checking! Added.